autoblocks/_impl/tracer.py

```diff
@@ -150,6 +217,8 @@ def send_event(
         parent_span_id: Optional[str] = None,
         timestamp: Optional[str] = None,
         properties: Optional[Dict] = None,
+        evaluators: List[BaseEventEvaluator] = [],
```

Comment:
mutable default args are not recommended in python, can you default to None and then turn it into a list:

```
evaluators: Optional[List[BaseEventEvaluator]] = None,

...

evaluators=evaluators or [],
```